spinta/datasets/backends/sql/components.py

```diff
@@ -12,6 +12,9 @@
 
 class Sql(ExternalBackend):
     type: str = 'sql'
+    accept_types = {
+        'text/xml',
```

Comment:
XML for SQL backend? I think, currently `Sql` backend does not support any direct upload formats, because this backends can only read data from database, and you can't upload a database, you could upload a database dump, bud `Sql` backend does not support that.